README.md

```diff
@@ -85,12 +85,12 @@ To use these scripts, please follow the following steps:
 6. **Push feedback to student's repository**
 
    ```shell
-   # If your feedback is in 'feedback.txt', run this command in the root directory.
+   # If your feedback is in 'feedback.md', run this command in the root directory.
    
-   ./push_feedback.sh feedback.txt
+   ./push_feedback.sh feedback.md
    
    # This will go through all repos in the subdirectories (check directory structure above), 
-   # push every file named 'feedback.txt' to corresponding repositories, 
+   # push every file named 'feedback.md' to corresponding repositories, 
    # so to name all feedback files with the same name is a good move.
    
```

Comment:
Thanks! It's useful to put it here too, but I think here any file name would work. I meant the marking scripts themselves.. (in the other repo) to have them generate feedback.md instead of feedback.txt